### PR TITLE
Update ug

### PR DESCRIPTION
**Pull request by oscarsu97**
Added a Reposense jar file as well

### CODE REVIEW

File: _reposense/config.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
       "githubId": "oscarsu97",
       "displayName": "OSCAR ... HONG",
       "authorNames": [
-        "oscarsu97", "Oscar", "Oscar Su"
+        "oscarsu97", "Oscar", "Oscar Su Zhen Hong"
```
Comment (1nefootstep) [inline _reposense/config.json]: no problem here but can you help me edit my name github name from `1newfootstep` to `1nefootstep`

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -212,47 +214,56 @@ Adds 7 days to the back of the itinerary.
 
 === Deleting days: `delete day`
 
-Deletes DAY_NUMBER th day of the itinerary. +
-Format: `delete day DAY_NUMBER`
+Deletes day DAY_INDEX of the itinerary. +
+Format: `delete day DAY_INDEX`
```
Comment (1nefootstep) [inline docs/UserGuide.adoc]: did we remove the prefix?

File: _reposense/config.json (hunk 1 above)
Comment (oscarsu97) [reply _reposense/config.json]: Sure👍

File: docs/UserGuide.adoc (hunk 2 above)
Comment (oscarsu97) [reply docs/UserGuide.adoc]: yes, the format will be updated to the ones above.

**Pull request merged**